index.js

```diff
@@ -453,6 +456,11 @@ class Hypercore extends EventEmitter {
     this.emit('close', true)
   }
 
+  async commit (state, opts) {
```

Comment:
should we make this commit(seesion, opts) while we are at it and sniff the state from the session?